### PR TITLE
Webfaqupdates

### PR DESCRIPTION
**Pull request by mariamhas**
@sfshaza2 please review, Thank you!

### CODE REVIEW

File: src/docs/development/platform-integration/web.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ No.
 
 ### Can I use Flutter plugins?
 
-Not yet.
+Yes, several plugins have web support. 
+Find an updated list of plugins on [pub.dev][] using the web filter.
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/web.md]: Can you provide a direct link to this filtered list?  Here's the link:  https://pub.dev/dart/packages?runtime=web

File: src/docs/development/platform-integration/web.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ No.
...
+You can also add web support to existing plugins or write your own plugins for the web.
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/web.md]: Where can they learn how to do this? I can't remember if we've added docs for this, or if there are any docs on the GitHub wiki.

Comment (mariamhas) [reply src/docs/development/platform-integration/web.md]: We only have a medium article currently. should i link to that? https://medium.com/flutter/how-to-write-a-flutter-web-plugin-5e26c689ea1

Comment (sfshaza2) [reply src/docs/development/platform-integration/web.md]: Yes, please! That's right. I reviewed that article.

**Comment by sfshaza2** [comment]
Also, I filed a bug against pub.dev. That web list should be a lot clearer that it IS a web list.

https://github.com/dart-lang/pub-dev/issues/3224

File: src/docs/development/platform-integration/web.md (hunk 1 above)
Comment (legalcodes) [reply src/docs/development/platform-integration/web.md]: @sfshaza re: the link you provided: aren't those dart packages?

Comment (mariamhas) [reply src/docs/development/platform-integration/web.md]: i think packages and plugins are mixed... i went on pub.dev and filtered by flutter and web to get this list : https://pub.dev/flutter/packages?platform=web 

which seems accurate to me as plugins

**Comment by sfshaza2** [comment]
Ahhh! OK, use that.

File: src/docs/development/platform-integration/web.md (hunk 1 above)
Comment (sfshaza2) [reply src/docs/development/platform-integration/web.md]: Great!

**Pull request merged**